woocommerce/woocommerce-android
# [Scan to update inventory] Part 5 - analytics events

Closes: #10338 


### Description


This PR adds the tracking events for the Scan To Update Inventory Project.

### Testing instructions


- Go to Product list screen, and tap the “Scan barcode” icon —  make sure `product_list_product_barcode_scanning_tapped ` is tracked
- Scan a barcode that is not registered on your store  — make sure `product_search_via_sku_failure` with source property `scan_to_update_inventory` is tracked. 
 - Scan an existing product - make sure `product_search_via_sku_successwith` the source property `scan_to_update_inventory` is tracked
 - Scan an existing product, then click on the button to increment the quantity - make sure  `product_quick_inventory_update_increment_quantity_tapped` is tracked
 - scan an existing product, then increase the quantity manually and tap on the update quantity button. Make sure `product_quick_inventory_update_manual_quantity_update_tapped` is tracked
 - Scan an existing product. Dismiss the bottom sheet. Make sure `product_quick_inventory_update_dismissed` is tracked. 
 - Scan existing product. Update quantity and trigger failure. Make sure `product_quick_inventory_quantity_update_failure` is tracked
 - Scan existing product. Tap on `view product details`  button. Make sure `product_quick_inventory_view_product_details_tapped` is tracked. 
 - after the quantity is successfully updated, make sure `product_quick_inventory_quantity_update_success` is tracked
### Images/gif
